src/common/models/cluster/cluster.ts

```diff
@@ -82,6 +84,7 @@ function ensureNotTiny(v: number): void {
 export class Cluster extends BaseImmutable<ClusterValue, ClusterJS> {
   static TYPE_VALUES: SupportedType[] = ['druid', 'mysql', 'postgres'];
   static DEFAULT_TIMEOUT = 40000;
+  static DEFAULT_HEALTH_CHECKING_TIMEOUT = 1000;
```

Comment:
There is no continuous form in other configuration properties. So I would use default_health_check_timeout.